src/ducks/apps/components/ApplicationPage/Header.jsx

```diff
@@ -41,11 +43,22 @@ export const Header = ({
     openApp(webviewIntent, app)
   }
 
-  const openConnector = () => {
+  const openConnector = async () => {
     if (isFlagshipApp()) {
       return webviewIntent.call('openApp', app.related, app)
     } else {
       const intents = new Intents({ client })
+      try {
+        // `intents.createService` throw an error if executed outside of an Intent iframe or if it is called without intentId.
+        const service = await intents.createService()
+
+        if (intentData.data.terminateIfInstalled) {
+          return service.terminate(app)
```

Comment:
can you add this `terminateIfInstalled` to the manifest please? In order to document it.

Reply:
Of course I was doing it